vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -309,3 +318,49 @@ def apply(self,
         if scheme is None:
             raise ValueError("A scheme must be defined for each layer")
         return scheme.apply_weights(layer, x, bias=bias)
+
+
+class CompressedTensorsKVCacheMethod(BaseKVCacheMethod):
+    """
+    Supports loading kv-cache scaling factors from compressed-tensors
+    checkpoints.
+    """
```

Comment:
Is this class completed? The only difference between this and Fp8KVCacheMethod seems just an additional validator?

Reply:
Yeah it's just a start of structure for future work. We could support integer or other scaling schemes within each method, so wanted a base class to bind them to.